docs/general/emulator-support-and-issues.md

```diff
@@ -326,7 +328,9 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 
 ### Nintendo 3DS
 
-- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ *Not supported* - needs hashing method and memory map
+- ❌ Standalone emulator: **Citra**
+  - No longer in active development due to Nintendo making the original creators take it down along with Yuzu. However, an active fork exists, made by PabloMK7 of CTGP-7 fame.
```

Comment:
We shouldn't wade into the business of commenting on these matters. I really don't want us getting search traffic from these keywords. Please revert.